src/AspNetCoreModuleV2/InProcessRequestHandler/precomp.hxx

```diff
@@ -40,6 +40,10 @@
 #undef WINVER
 #undef _WIN32_WINNT
 
+#include <acache.h>
+#include <exceptions.h>
```

Comment:
Add these to individual classes.

Reply:
those two will be added for almost every file. Not a good idea to duplicate the include everywhere